README.md

```diff
@@ -331,4 +344,11 @@ The `outbound-edi` function can be invoked via the UI for testing.
 
   </details>
 
-5. You can view the file using the [Buckets Web View](https://www.stedi.com/app/buckets). As shown above, the output of the function includes the `bucketName` and `key` (path within the bucket) of where the generated EDI was saved.
\ No newline at end of file
+5. You can view the file using the [Buckets UI](https://www.stedi.com/app/buckets). As shown above, the output of the function includes the `bucketName` and `key` (path within the bucket) of where the generated EDI was saved.
+
+# Customizing the Bootstrap Workflow
+
+The bootstrap workflow uses a sample trading partner to set up and test the read and write EDI workflows. You can customize the bootstrap workflow for your use case by doing one or all of the following:
+- [Add a partner profile](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#add-a-trading-partner-profile) to replace the test trading partner with your real trading partners' details and requirements.
+- [Create a Stedi mapping](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#map-inbound-messages) that transform EDI documents into a custom JSON shape for your internal system. The base bootstrap repository ingests and generates JSON with a schema that closely matches EDI documents. 
```

Comment:
another nit: may want to generalize the use case here, as mapping can be in either direction between guide JSON and a custom API (not necessarily just EDI-like JSON to custom API, could be the other way around too)

Reply:
> nit: s/common read and write EDI use case/common read and write EDI use cases/

Depends on whether you see reading and writing EDI as a single use case or as two.

> I'm not sure about this phrasing: "design a custom integration."

We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you customize the workflow.

Although, what you wrote, works too, Rob. @Laura, now you have options. :smile: